Apps/W1/Shopify/app/src/Catalogs/Pages/ShpfyCatalogs.Page.al

```diff
@@ -25,6 +25,11 @@ page 30159 "Shpfy Catalogs"
                     ToolTip = 'Specifies the unique identifier for the catalog in Shopify.';
                     Editable = false;
                 }
+                field("Customer No."; Rec."Customer No.")
```

Comment:
Visible = false, please.

Extend tooltip, to highlight that parameters defined on the customer card, such as Cust Disc Group, Cust Price Group, Allow Line Disc have priority over values defined in the catalog.

Reply:
@AndreiPanko If this is marked as visible false should customer extend this app to make it visible again

Reply:
No, they will be able to do it via personalization. No development is needed.